src/components/pager/pager.stories.js

```diff
@@ -68,7 +68,7 @@ storiesOf('Pager', module)
     }
   })
   .add('default', () => {
-    const totalRecords = text('totalRecords', '100');
+    const totalRecords = number('totalRecords', '100');
```

Comment:
Presumably this works, but should 100 not be a number?

Reply:
Agreed, fixed 👍